homeassistant/components/playstation_network/config_flow.py

```diff
@@ -176,6 +176,8 @@ async def async_step_user(
     ) -> SubentryFlowResult:
         """Subentry user flow."""
         config_entry: PlaystationNetworkConfigEntry = self._get_entry()
+        if config_entry.disabled_by:
+            return self.async_abort(reason="config_entry_disabled")
```

Comment:
Wouldn't this be something that Home Assistant itself should account for? I would assume that you can't add sub entries to a disabled entry

Reply:
I'll check back with Joost about this.

In general, it's not obvious that adding a subentry to a disabled entry should not be possible.
If we don't want to allow it, it's not clear we should block it in core, or just not enable the "Add xxx" option in frontend.